docs/reference/transform/apis/get-transform-stats.asciidoc

```diff
@@ -83,11 +83,34 @@ informational; you cannot update their values.
 `checkpointing`.`last`:::
 (object) Contains statistics about the last completed checkpoint. 
 `checkpointing`.`last`.`checkpoint`::::
-(TBD) A unique identifier for the checkpoint.
+(integer) The sequence number for the checkpoint.
 `checkpointing`.`last`.`time_upper_bound_millis`::::
-(date) TBD
+(date) When using time-based synchronization, this timestamp indicates the
+upper bound of data that is included in the checkpoint. Typically, this value
+is equal to the `checkpointing`.`last`.`time_upper_bound_millis` minus the
+`sync`.`time`.`delay`, which is defined when you create the {transform}.
 `checkpointing`.`last`.`timestamp_millis`::::
-(date) TBD
+(date) The timestamp of the checkpoint, which indicates when the checkpoint
+was created.
+`checkpointing`.`next`:::
+(object) Contains statistics about the next checkpoint that is currently in
+progress. This object appears only if the {transform} is currently processing
+data and only for the first checkpoint.
```

Comment:
sorry this is not quite correct yet: "and only for the first checkpoint" is only true for the `checkpoint_progress` nested object below

Reply:
Thanks for the clarifications!  I've pushed another commit